__tests__/app.test.js

```diff
@@ -78,291 +78,312 @@ describe("app", () => {
-        test("Status 200: Should update article with correct votes", () => {
-          return request(app)
-            .patch("/api/articles/1")
-            .send({ inc_votes: 1 })
-            .expect(200)
-            .then(({ body }) => {
-              const { article } = body;
-              expect(article).toEqual({
-                article_id: 1,
-                title: "Living in the shadow of a great man",
-                topic: "mitch",
-                author: "butter_bridge",
-                body: "I find this existence challenging",
-                created_at: "2020-07-09T20:11:00.000Z",
-                votes: 101,
-                article_img_url:
-                  "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
-              });
-            });
-        });
-        test("Status 200: Should update article with correct vote if inc_votes is negative", () => {
-          return request(app)
-            .patch("/api/articles/1")
-            .send({ inc_votes: -100 })
-            .expect(200)
-            .then(({ body }) => {
-              const { article } = body;
-              expect(article).toEqual({
-                article_id: 1,
-                title: "Living in the shadow of a great man",
-                topic: "mitch",
-                author: "butter_bridge",
-                body: "I find this existence challenging",
-                created_at: "2020-07-09T20:11:00.000Z",
-                votes: 0,
-                article_img_url:
-                  "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
-              });
-            });
-        });
-        test("Status 404: Should return not found if article id is valid but incorrect", () => {
-          return request(app)
-            .patch("/api/articles/999")
-            .send({ inc_votes: 1 })
-            .expect(404)
-            .then((res) => {
-              expect(res.body.msg).toBe(`No article found for article: 999`);
-            });
-        });
-        test("Status 400: Should return Bad request if article id is not valid", () => {
-          return request(app)
-            .patch("/api/articles/not-an-article")
-            .send({ inc_votes: 1 })
-            .expect(400)
-            .then((res) => {
-              expect(res.body.msg).toBe(`Bad request`);
-            });
-        });
-        test("Status 400: Should return Bad request if body has a string instead of a number", () => {
-          return request(app)
-            .patch("/api/articles/1")
-            .send({ inc_votes: "string" })
-            .expect(400)
-            .then((res) => {
-              expect(res.body.msg).toBe(`Bad request`);
+    });
+
+    describe("GET /api/articles/:article_id", () => {
+      test("Status 200: Should respond with an article object using the id and now with a comment count", () => {
```

Comment:
You could also refactor the test from a previous ticket which makes sure the article object looks as expected, and just add a line there to ensure comment count is also on it 🙂